cime_config/SystemTests/lciso.py

```diff
@@ -0,0 +1,36 @@
+"""
+Implementation of the CIME LCISO (Land Carbon Isotope) test.
+
+This is a CTSM specific test:
+Verifies turning on carbon isotopes doesn't change answers
+(1) do a run with Carbon isotopes off (suffix base)
+(2) add C13 and C14 carbon isotopes on with their time-series (suffix cisoallon)
+
+"""
+
+from CIME.SystemTests.system_tests_compare_two import SystemTestsCompareTwo
+from CIME.XML.standard_module_setup import *
+from CIME.SystemTests.test_utils.user_nl_utils import append_to_user_nl_files
+
+logger = logging.getLogger(__name__)
+
+class LCISO(SystemTestsCompareTwo):
+
+    def __init__(self, case):
+        SystemTestsCompareTwo.__init__(self, case,
+                                       separate_builds = False,
+                                       run_two_suffix = 'cisoallon',
+                                       run_one_description = 'carbon isotopes off',
+                                       run_two_description = 'c13 and c14 isotopes on as well as C isotope time series',
+                                       ignore_fieldlist_diffs = True)
+
+    def _case_one_setup(self):
+        append_to_user_nl_files(caseroot = self._get_caseroot(),
+                                component = "clm",
+                                contents = "use_c13=F, use_c14=F")
```

Comment:
Does this actually work? I thought you needed to set these to `.true.` or `.false.`, not `F`?

Reply:
Yes, I could check which version of the FORTRAN standard added it, but it is now in the standard and works on our machines and compilers and in our perl and python scripts. The one caveat there is that, in the namelist_defaults files we sometimes explicitly check for ".true." or ".false." rather than "T" or "F", but if you add the appropriate logic using the check for the IsThisTrue function it will I know I don't have the name right). The FORTRAN standard allows all of these...

.true.
.TRue.
T
F
I think it allows shortened versions as well .TRU., but not sure. But, the logical function checks for the valid forms.

Reply:
Okay, thanks, I didn't realize this. The asymmetry here (`F` and `.true.` a few lines later) still offends my sensibilities a tiny bit (I guess I'm a sensitive guy), but it's not a big deal :-)

Reply:
Ah, yeah I had to have that there because of the issue with build-namelist sometimes searching for the exact ".true." string. So it perhaps should be consistent. I like the being short and concise that we get with the T and F.

Reply:
Leaving as is. I should maybe create an issue about the build-namelist problem of doing matching on the exact ".true." string in namelist defaults.